catalog/gke-autopilot/Kptfile

```diff
@@ -0,0 +1,13 @@
+apiVersion: kpt.dev/v1
```

Comment:
This blueprint should be derivative of the existing GKE blueprint (ie. upstream should be GKE). You first step for build this blueprint should have been to run `kpt pkg get`.

Reply:
That is how I started but I removed the upstream attribution to make it standalone. I will add that back in.

Reply:
Kpt file updated

Reply:
Upstream is still not present. Please push!

Reply:
Pushed the changes